core/node/fee_model/src/l1_gas_price/gas_adjuster/metrics.rs

```diff
@@ -7,9 +7,11 @@ use vise::{Gauge, Metrics};
 pub(super) struct GasAdjusterMetrics {
     pub current_base_fee_per_gas: Gauge<u64>,
     pub current_blob_base_fee: Gauge<u64>,
+    pub current_l2_pubdata_price: Gauge<u64>,
     pub median_base_fee_per_gas: Gauge<u64>,
     pub median_blob_base_fee_per_gas: Gauge<u64>,
     pub median_blob_base_fee: Gauge<u64>,
+    pub median_l2_pubdata_price: Gauge<u64>,
```

Comment:
This metric doesn't seem to be used.

Reply:
Not sure what would be the best place to use it, I'll delete it for now